bundle/config/resources/secret_scope.go

```diff
@@ -0,0 +1,73 @@
+package resources
+
+import (
+	"context"
+	"net/url"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/marshal"
+	"github.com/databricks/databricks-sdk-go/service/workspace"
+)
+
+type SecretScope struct {
+	Name                   string `json:"name"`
+	InitialManagePrincipal string `json:"initial_manage_principal"`
```

Comment:
Is this ever different from the calling principal?

Reply:
as decided - we won't have this field in DABs